Assets/MRTK/Core/Utilities/Gltf/Schema/GltfAccessor.cs

```diff
@@ -31,7 +32,10 @@ public class GltfAccessor : GltfChildOfRootProperty
         /// 5125 (UNSIGNED_INT) is only allowed when the accessor contains indices
         /// i.e., the accessor is only referenced by `primitive.indices`.
         /// </summary>
-        public GltfComponentType componentType;
+        public GltfComponentType ComponentType { get; set; }
```

Comment:
The Private and public fields do not seem connected.  Was this intentional or has something been missed?

Reply:
Ok, I think I see your reasoning, separating the serialization from the access via enum.

Reply:
Unity's deserializer doesn't handle enums the same way the glTF spec does. The spec serializes them by string, while Unity serializes them by int value, so this has been broken since it was first contributed.

These are connected via the `ISerializationCallbackReceiver` events below.